tests/sentry/workflow_engine/endpoints/test_organization_available_action_index.py

```diff
@@ -0,0 +1,407 @@
+        self.webhooks_plugin = self.plugin_registry.get(WebHooksPlugin.slug)
+        self.webhooks_plugin.enable(self.project)
+
+        self.plugin_registry.register(SlackPlugin)
+        self.slack_plugin = self.plugin_registry.get(SlackPlugin.slug)
+        self.slack_plugin.enable(self.project)
+
+        # non notification plugins should not be returned
+        self.plugin_registry.register(TrelloPlugin)
+        self.trello_plugin = self.plugin_registry.get(TrelloPlugin.slug)
+        self.trello_plugin.enable(self.project)
+
+        # plugins that are not enabled should not be returned
+        self.plugin_registry.register(PagerDutyPlugin)
+
+        response = self.get_success_response(
+            self.organization.slug,
+            status_code=200,
+        )
+        assert len(response.data) == 1
+        assert response.data == [
+            {
+                "type": Action.Type.WEBHOOK,
+                "handlerGroup": ActionHandler.Group.OTHER.value,
+                "configSchema": {},
+                "dataSchema": {},
+                "services": [
+                    {"slug": "slack", "name": "(Legacy) Slack"},
+                    {"slug": "webhooks", "name": "WebHooks"},
+                ],
+            }
+        ]
+
+    def test_actions_sorting(self):
+        @self.registry.register(Action.Type.MSTEAMS)
+        @dataclass(frozen=True)
+        class MSTeamsActionHandler(IntegrationActionHandler):
+            group = ActionHandler.Group.NOTIFICATION
+            provider_slug = "msteams"
+            config_schema = {}
+            data_schema = {}
+
+        @self.registry.register(Action.Type.EMAIL)
+        @dataclass(frozen=True)
+        class EmailActionHandler(ActionHandler):
+            group = ActionHandler.Group.NOTIFICATION
+            config_schema = {}
+            data_schema = {}
+
+        @self.registry.register(Action.Type.SENTRY_APP)
+        @dataclass(frozen=True)
+        class SentryAppActionHandler(ActionHandler):
+            group = ActionHandler.Group.OTHER
+            config_schema = {}
+            data_schema = {}
+
+        self.no_component_sentry_app = self.create_sentry_app(
+            name="Poppy's Fire Sentry App",
+            organization=self.organization,
+            is_alertable=True,
+        )
+        self.no_component_sentry_app_installation = self.create_sentry_app_installation(
+            slug=self.no_component_sentry_app.slug, organization=self.organization
+        )
+
+        self.sentry_app = self.create_sentry_app(
+            name="Moo Deng's Fire Sentry App",
+            organization=self.organization,
+            schema={
+                "elements": [
+                    self.create_alert_rule_action_schema(),
+                ]
+            },
+            is_alertable=True,
+        )
+        self.sentry_app_installation = self.create_sentry_app_installation(
+            slug=self.sentry_app.slug, organization=self.organization
+        )
```

Comment:
Could make this slightly dry-er by making a `create_sentry_apps()` function to call here and in `test_sentry_apps`